### PR TITLE
Markdown conversion for `pt-BR` - Replace - HTML section :warning:  Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of #7949
Conversion report: https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-11t09_08_07-506z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning: 

@mdn/yari-content-pt-br / @josielrocha Could you please check this one in the coming days as conflicts are very likely to arise / this PR is liely to be stale if other edits occur on the same subset of pages. Thank you :pray:

File: files/pt-br/web/html/element/blockquote/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - blogger
+  - português brasil
+  - tag
+translation_of: Web/HTML/Element/blockquote
+---
+## Sumário
+
+O Elemento HTML \<blockquote> (ou Elemento HTML de citação de bloco) indica que o texto incluído é uma longa citação. Normalmente, este é processado visualmente pelo recuo (ver [Notas](/pt-BR/docs/HTML/Element/blockquote#Notes) sobre como mudá-lo). A URL para a fonte da citação pode ser dada usando o atributo **cite**, enquanto uma representação de texto da fonte pode ser dada usando o {{HTMLElement("cite")}} elemento.
+
+-
```
Comment (josielrocha) [inline files/pt-br/web/html/element/blockquote/index.md:19]:
```suggestion
```

**Review by josielrocha — COMMENTED** [review]
Thank you for your hard work @SphinxKnight!

I made a comment in one file to remove a `-` character that as generating an empty list item.

Could you please fix it?

**Review by josielrocha — APPROVED** [review]
LGTM ;)

Thank you @SphinxKnight!

**Pull request merged**